### PR TITLE
feat(): Add the missing options to TATUI

### PR DESCRIPTION
**Pull request by asturur**
## Description

Enable/Disable for exit_game works
Detecting and entering exit_game_delay works, and validates the value being a positive number
Added Andrea Bogazzi to the about screen
Incremented version of tapui to 0.6

### CODE REVIEW

**Comment by sigboe** [comment]
LGTM so far, I haven't tested it, but its very close to what I would have done.
I think this can be merged so far, if you remove the comment with the half finished `_exitGameDelaySetting()` function

File: scripts/taptui/taptui.sh [hunk 1]
```diff
@@ -998,6 +998,99 @@ _probeSetting() {
...
+      while true; do
+        delayInSeconds="$(_inputbox "Enter delay in seconds" "${customString}")"
+        exitcode="${?}"; [[ "${exitcode}" -ge 1 ]] && { "${FUNCNAME[0]}" ; return ; }
+        [[ "${delayInSeconds}" == +([0-9]*) ]] && break
+        _error "${delayInSeconds} is not a positive number"
+      done
+      if grep -q "^exit_game_delay=" "${settings}"; then
+        sed -i "s/^exit_game_delay=.*/exit_game_delay=${delayInSeconds}/" "${settings}"
+      else
+        echo "connection_string=${delayInSeconds}" >> "${settings}"
```
Comment (asturur) [inline scripts/taptui/taptui.sh:1088]: @sigboe ah ahhhh and this error?

**Comment by asturur** [comment]
I copied the game list from the connection string, there is no way i can pull in the core list and other things i m not sure we want to do.
Is manual. period.

let me know, the last commit is untested and i won't be able to test it before few hours

**Comment by asturur** [comment]
```bash
  if grep -q "^exit_game_blocklist=\".+\"" "${settings}"; then
    menuOptions[5]="on"
  else 
    menuOptions[2]="on"
  fi
```
i was expecting this with the `+` to detect non empty strings, but i can't make it work

**Pull request merged**